deepl/deepl.py

```diff
@@ -131,9 +131,11 @@ async def __translate(self, script: str) -> str:
                 # banner prevents clicking on language buttons, close the banner first
                 await page.click("button[data-testid=cookie-banner-lax-close-button]")
                 # select input / output language
-                await page.click("button[data-testid=translator-source-lang-btn]")
+                await page.click("button[data-testid=translator-source-lang-btn]", delay=1000)
```

Comment:
I dislike this because it makes each translation 1s slower than now. Please add the reason to PR description.